utils/translation_service.py

```
@@ -0,0 +1,251 @@
+        except Exception as e:
+            logger.error(f"翻译服务：初始化LLM客户端失败: {e}")
+            return None
+    
+    def _get_from_cache(self, text: str, target_lang: str) -> Optional[str]:
+        """从缓存获取翻译结果"""
+        cache_key = self._get_cache_key(text, target_lang)
+        return self._cache.get(cache_key)
+    
+    def _save_to_cache(self, text: str, target_lang: str, translated: str):
+        """保存翻译结果到缓存"""
+        # 简单的FIFO缓存：如果缓存过大，删除最早加入的条目
```

Ruff (0.14.8)

Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)